config/routes.rb

```diff
@@ -1,9 +1,17 @@
 Rails.application.routes.draw do
-  root "users#index"
+
+  devise_for :users
+
+  devise_scope :user do
+    get '/users/sign_out' => 'devise/sessions#destroy'
+  end
```

Comment:
- [ ] I can see you found a way around the delete `link_to` issue, kudos 👍🏻 
You can also solve that by making `delete` a `button_to`